### DISCUSSION TITLE
Options select

### DISCUSSION THREAD

**Discussion by rodrigocipriani** [iSoron/uhabits#1333] [Feature Requests]
Sometimes we try our best, but we don't achieve the habit as we want, for example, eat heathy, today I ate quite well so it's no yes or no, it's or something like that. Another example could be to exercise, sometimes we don´t achieve what we wanted, but was not so bad. 
I think this kind of track helps you to at least do the habit, even if it's not as you plan.

_In the app we have measurable habits, I've tried to use, but it doesn't work as well._

### More practical examples:

**Eating Heatly:** Options (Done, almost there, not so bad, Not done, Skip)
**Doing exercise:** Options (⭐ - 1,2,3,4,5)
**Run** (1km, 5km, 10km, 0km)

Something like that. 
The "Run" example is not so good. But the other ones I've used in a excel habit I made, and it was especially useful 👌

**Other discussion like this:**
https://github.com/iSoron/uhabits/discussions/1267
https://github.com/iSoron/uhabits/discussions/979

**Comment by rodrigocipriani** [comment]
If this feature seems nice to the app, I can try to work on this. Just let me know.

**Comment by spartan** [comment]
Hello,

This looks quite similar to what I would like from the app:
For example for a task like "Workout" we can choose from a list of options defined by myself:
- upper-body
- lower-body
- core
This would help to avoid creating multiple other tasks for each of the option.

What do you think?

**Comment by iSoron** [comment]
Looks like a duplicate of #409
